src/OrchardCore.Modules/OrchardCore.Security/Options/PermissionsPolicyValue.cs

```diff
@@ -0,0 +1,57 @@
+using Microsoft.Extensions.Primitives;
+
+namespace OrchardCore.Security.Options
+{
+    public class PermissionsPolicyValue
```

Comment:
Isn't that called an enum? What concerns me is all these conversions from string to class to string again for a `Name` property. It's either just a string from beginning to end or an enum if you really don't want strings.

Reply:
The only limitation in enum is they have a numeric values, unless you add some annotations in enum values, the intent here is to use strongly-typed names and values behind the scene

Reply:
We could use them as string from begging to end, which is I did few days ago, but this might need some docs on APIs to let the people knows what are the enum-like class which provides the valid values

Reply:
Maybe as we do for the custom PredefinedList editor for TextField, at the end this is a text but on the front end a drop down only listing possible values.

Reply:
You are right Jean-Thierry, my intend from the beginning is to make the APIs clear, if we use them outside OC, I will revert my changes ...